src/plugins/europeana/record.js

```diff
@@ -301,8 +312,11 @@ export default (context = {}) => {
     * @return {Boolean} true if enriched data will be shown
     */
     localeSpecificFieldValueIsFromEnrichment(field, providerProxy, proxies, predictedUiLang) {
-      if (providerProxy[field][predictedUiLang] ||
-        (!proxies[2][field][predictedUiLang] && providerProxy[field]['en'])) {
+      if (isLangMap(providerProxy[field]) &&
+           (providerProxy[field]?.[predictedUiLang] ||
+             (!proxies[2][field]?.[predictedUiLang] && providerProxy[field]?.['en'])
```

Comment:
Should this still be checking `'en'`?

Reply:
yes.

This catches the case where:

- The record does not have a recognized/supported EDM language, meaning it will use the 'old' language prioritization
- There is no exact match on the enriched (provider) proxy
- The main (last index) proxy contains no exact language match
- Meaning the fallback logic will kick in.
- The enriched (provider) proxy contains an 'en' value that will be displayed due to fallback being active.